plugins/procedural/main.cpp

```diff
@@ -46,10 +46,13 @@ inline ProceduralReader *CreateProceduralReader(AtUniverse *universe)
 {
 #ifdef ENABLE_HYDRA_IN_USD_PROCEDURAL
     if (ArchHasEnv("PROCEDURAL_USE_HYDRA")) {
-        return new HydraArnoldReader(universe);
-    } else {
-        return new UsdArnoldReader();
+        const std::string useHydra = ArchGetEnv("PROCEDURAL_USE_HYDRA");
+        int envVal = std::stoi(useHydra);
```

Comment:
why not testing the string directly ? the code might raise an exception if PROCEDURAL_USE_HYDRA is not convertible to int

Reply:
in my first test after setting the variable in command line, the string was "0 " and not "0" as I was expecting. So I thought this would be better than stripping empty characters. But we can do that too